runtime/drivers/duckdb/transporter_duckDB_to_duckDB_test.go

```diff
@@ -31,7 +31,7 @@ func TestDuckDBToDuckDBTransfer(t *testing.T) {
 	require.NoError(t, err)
 	require.NoError(t, conn.Close())
 
-	to, err := Driver{}.Open("default", map[string]any{"dsn": ":memory:"}, activity.NewNoopClient(), zap.NewNop())
+	to, err := Driver{}.Open("default", map[string]any{"path": filepath.Join(tempDir, "main.db")}, activity.NewNoopClient(), zap.NewNop())
```

Comment:
Should it be `tempDir, "tmp", "main.db"`? Worried this might cause `main.db` to be read in repo listings or trigger watches in the project parser, which could lead to some overhead or weird triggers

Reply:
So we are not starting any runtime in this test. It directly opens a duckdb handle and executes transfer so it should be okay here ?

Reply:
Ah okay, I see, then it's fine!